lib/bolt/cli/src/commands/admin/cluster/datacenter.rs

```diff
@@ -0,0 +1,142 @@
+use anyhow::*;
+use bolt_core::context::ProjectContext;
+use clap::{Parser, ValueEnum};
+use rivet_api::{
+	apis::{configuration::Configuration, *},
+	models,
+};
+
+#[derive(ValueEnum, Clone)]
+pub enum DatacenterAdminProvider {
```

Comment:
is "admin" required in the name since it's already in the context of `admin::cluster`? same applies for all other types.